tests/examples/custom/custom.tf

```diff
@@ -1,8 +1,4 @@
-variable "count" {
-    default = "2"
-}
-
-variable "network" {
+variable "net_octet" {
```

Comment:
this example of custom config. maybe it would be nice to leave it with network variable to show "another" way ?

Reply:
not sure I get your point. You want to hardcode the network?  
I also changed the var network->net_octet in default.tf, which is the main point of this PR.

Reply:
yes I do . this is the point of having "custom" setup to show that you can do things in different way, right ? :)

Reply:
but obviously it is not something to block merge of this PR , you can keep it like this too